jobs/delorean/2.0/ews/3scale/3scale.yaml

```diff
@@ -0,0 +1,34 @@
+---
+- job:
+    name: 3scale_brew_build_created
```

Comment:
Prepend this with delorean-2-ews/3scale/ "delorean-2-ews/3scale/3scale_brew_build_created"

Reply:
We probably want to consider what this should actually be named as well, previously we just called the equivalent 1.0 job discovery.yaml https://github.com/integr8ly/ci-cd/blob/master/jobs/delorean/1.0/ews/3scale/ga/discovery.yaml